lib/__tests__/VCard.js

```diff
@@ -46,9 +47,9 @@ TEL;WORK:123456789\r\n\
 ADR;WORK;POSTAL;CHARSET=utf-8:name;extended;street;worktown;state;workpos\r\n\
  tcode;Belgium\r\n\
 URL:http://www.jeroendesloovere.be\r\n\
+PHOTO:MEDIATYPE=image/gif:http://www.example.com/dir_photos/my_photo.gif\r\n\
```

Comment:
```suggestion
PHOTO;VALUE=uri;TYPE=gif:http://www.example.com/dir_photos/my_photo.gif\r\n\
```

I think this should be the vCard 3.0 format output.